web/src/i18n.js

```diff
@@ -0,0 +1,46 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+import LanguageDetector from 'i18next-browser-languagedetector';
+
+i18n
+  // detect user language
+  // learn more: https://github.com/i18next/i18next-browser-languageDetector
+  .use(LanguageDetector)
+  // pass the i18n instance to react-i18next.
+  .use(initReactI18next)
+  // init i18next
+  // for all options read: https://www.i18next.com/overview/configuration-options
+  .init({
+    debug: true,
+    fallbackLng: 'en',
+    interpolation: {
+      escapeValue: false, // not needed for react as it escapes by default
+    },
+    resources: {
+      en: {
+        translation: {
+          // here we will place our translations...
+          header: 'API Docs',
+          JobTab1: 'LATEST RUN',
+          JobTab2: 'RUN HISTORY'
+        }
+      },
+      fr: {
+        translation: {
+          header: 'API Docs',
+          JobTab1: 'DERNIÈRE COURSE',
+          JobTab2: "HISTORIQUE D'EXECUTION"
+        }
+      },
+      sp: {
+        translation: {
+          header: 'API Docs',
+          JobTab1: 'ÚLTIMA EJECUCIÓN',
+          JobTab2: 'HISTORIAL DE EJECUCIONES'
+        }
+      }
+
+    }
+  });
+
+export default i18n;
```

Comment:
You can apply formatting by running `npm run eslint-fix` that will cleanup newline issues like this.

Reply:
ran eslint with npx, think I resolved this